camayoc/qpc_models.py

```diff
@@ -592,3 +593,75 @@ def equivalent(self, other):
         raise NotImplementedError(
             'ScanJobs do not have an equivalent() method.'
         )
+
+
+class Report(QPCObject):
+    """A class to aid in the creation and control of Reports."""
+
```

Comment:
Can you add into this doc string how you can get the report id from a scan?

Also example of how you might create a report object and use to merge results from multiple scans.